bin/welcome.js

```diff
@@ -0,0 +1,10 @@
+#! /usr/env/bin node
+
+const chalk = require( 'chalk' );
+
+console.log( chalk.cyan( '             _                           ' ) );
+console.log( chalk.cyan( '    ___ __ _| |_   _ _ __  ___  ___      ' ) );
+console.log( chalk.cyan( '   / __/ _\` | | | | | \'_ \\/ __|/ _ \\ ' ) );
+console.log( chalk.cyan( '  | (_| (_| | | |_| | |_) \\__ \\ (_) |  ' ) );
+console.log( chalk.cyan( '   \\___\\__,_|_|\\__, | .__/|___/\\___/ ' ) );
+console.log( chalk.cyan( '               |___/|_|                \n' ) );
```

Comment:
We can always open a follow-up to tackle all requests related to Calypso ascii art. It seems like it is the most discussed change here :)